src/components/ServiceDetails/MediaGallery/index.js

```diff
@@ -42,7 +41,7 @@ class MediaGallery extends Component {
 
     this.filteredData = this.props.data
       ? this.props.data.filter(item => {
-          return item.asset_type !== HERO_IMG;
+          return item.asset_type === "media_gallery";
```

Comment:
can we get the asset_type from an enum instead of hardcoding?
We already have an enum for this I guess.

Reply:
okay